controllers/postgres_controller.go

```diff
@@ -7,16 +7,21 @@
 package controllers
 
 import (
+	"bytes"
 	"context"
 	"encoding/json"
+	goerrors "errors"
```

Comment:
remove this alias and create alias to apierros in line 27

Reply:
renamed `errors` to `apierrors`, `goerrors` to `errors`, and used `corev1` instead of `v1` (which was the same import anyway)